lib/rubykassa/payment_interface.rb

```diff
@@ -48,14 +47,15 @@ def initial_options
         login: Rubykassa.login,
         total: @total,
         invoice_id: @invoice_id,
+        is_test: test_mode? ? 1 : 0,
         signature: generate_signature_for(:payment)
       }.merge(Hash[@params.sort.map {|param_name| [param_name[0], param_name[1]]}])
     end
 
     private
 
-      def shpfy_params
-        @params = @params.map {|param_name| ["shp#{param_name[0]}".to_sym, param_name[1]]}
-      end
+    def shpfy_params
+      @params = @params.map {|param_name| ["shp#{param_name[0]}".to_sym, param_name[1]]}
```

Comment:
Line is too long. [88/80]<br>Space between { and | missing.<br>Space missing inside }.